compact.go

```diff
@@ -471,7 +471,7 @@ func (w *instrumentedChunkWriter) WriteChunks(chunks ...chunks.Meta) error {
 	return w.ChunkWriter.WriteChunks(chunks...)
 }
 
-// write creates a new block that is the union of the provided blocks into dir.
+// write creates a new block into a directory by including data within the requested mint:maxt range.
```

Comment:
not even sure why we need this comment it is a complete copy of the comment for `Write(...`

Reply:
agree